ext/sodium/config.m4

```diff
@@ -1,5 +1,5 @@
 dnl $Id$
-dnl config.m4 for extension libsodium
+dnl config.m4 for extension sodium
 
 PHP_ARG_WITH(libsodium, for libsodium support,
 [  --with-libsodium[[=DIR]]  Include libsodium support])
```

Comment:
Sure that's what I meant. I was just replying in regards to the one of the options that you suggested. Basically it shouldn't be `enable` but `with`. In this case `--with-sodium` ofc.